### PR TITLE
feat: alert-button

### PR DESCRIPTION
**Pull request by yashpatel1797**
#1 Alert and button components are added.
#2 Code snippet design is not completed so ignore it for now.
[live link](https://thirsty-engelbart-dd2270.netlify.app/docs.html)

### CODE REVIEW

File: components/Button/button.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+}
+.btn-link-bg:hover {
+  background-color: var(--color-primary);
+  color: white;
+}
+.btn-float {
+  border-radius: 50%;
+  background-color: var(--color-primary);
+  color: var(--color-white);
+  padding: 0.7rem;
```
Comment (nyyshaaa) [inline components/Button/button.css]: You can increase padding to make it somewhat more like circle.
padding:10px 12px; it works fine according to this btn's height and width.

**Review by nyyshaaa — COMMENTED** [review]
code snippets you mentioned are incomplete, all else looks good. Also just in btn's  case their text font-size looks somewhat small .

File: components/Button/button.css (hunk 1 above)
Comment (yashpatel1797) [reply components/Button/button.css]: Thanks for the suggestion will change this.

**Comment by yashpatel1797** [comment]
> code snippets you mentioned are incomplete, all else looks good. Also just in btn's case their text font-size looks somewhat small .

I will increase the font size thanks for identifying this.

**Review by aneenajohn — CHANGES_REQUESTED** [review]
UI looks clean and neat
1) Be consistent across naming classes if you are using BEM it has to be used throughout.
2) Check the header and responsiveness on small screens.
3) Give primary color for the header(preferably) or something which would have some contrast from the background.
4) Add active state for the side navbar

**Pull request merged**